functions/memorystore/redis/index.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2019, Google LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// [START functions_memorystore_redis]
+
+const redis = require('redis');
+
+const REDISHOST = process.env.REDISHOST || 'localhost';
+const REDISPORT = process.env.REDISPORT || 6379;
+
+const redisClient = redis.createClient(REDISPORT, REDISHOST);
+redisClient.on('error', err => console.error('ERR:REDIS:', err));
+
+exports.visitCount = (req, res) => {
+  redisClient.incr('visits', (err, reply) => {
```

Comment:
Can we convert this to promises directly (if `incrAsync` is a supported function), or use [`util.promisify`](https://nodejs.org/dist/latest-v8.x/docs/api/util.html) here?

Reply:
Done :+1: